catalog/core/catalog-core-transform-impl/src/main/java/org/codice/ddf/catalog/locator/TransformerLocator.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.locator;
+
+import ddf.catalog.transform.QueryResponseTransformer;
+import java.util.List;
+import javax.activation.MimeType;
+import org.codice.ddf.catalog.transform.MultiInputTransformer;
+import org.codice.ddf.catalog.transform.MultiMetacardTransformer;
+
+/**
+ * <b> This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library. </b>
+ */
+public interface TransformerLocator {
```

Comment:
is there a reason this interface is in the impl?

Reply:
This interface is meant to be purely internal to the impl. We want to be able to swap out different TransformerLocator implementations, but we don't want people using the interface elsewhere in the system.